SR2019/2018-12-07-techday-announce.md

```diff
@@ -0,0 +1,22 @@
+---
+to: Student Robotics 2019
+subject: Announcing tech days
+---
+
+As was announced at kickstart, we intend to run some tech days over this competition year. Tech days are opportunities for teams to get help if they’re struggling with a technical issue, and is a controlled environment for students to make more progress on their robots. It’s also an opportunity to see how well they’re doing in comparison to other teams.
+
+As with all other events, students *must* be accompanied by a responsible adult.
+
+We will be running tech days at the University of Southampton on the following days:
+
+- 15th December 2018
+- 9th February 2019
+- 9th March 2019
+
+These tech days will be at the Highfield campus, start around 10am, and finish around 5pm. However please turn up and leave whenever's appropriate.
+
+We would like to also run some tech days in London. If you would be interested in a tech day in London too on these dates, please let us know as soon as possible. For the December tech day, we need enough teams by the 12th to make it happen.
+
+Specific details for these tech days will be available on our website soon.
```

Comment:
Maybe drop the "these" here? That allows it to imply that other tech days might also appear on the website too.